examples/overwrite-interpreter-options/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "config" : { "example_name" : "overwrite-interpreter-options" },
```

Comment:
Since you dont overwrite the interpreter options, maybe `passing-interpreter-options` is better 🤔 ?

Reply:
Good call, fixed.